npms/skale-observer/observer.js

```diff
@@ -232,6 +232,89 @@ async function load_schains( w3, addressFrom, opts ) {
     return arr_schains;
 }
 
+async function load_cached_schains_simplified( w3, addressFrom, opts ) {
```

Comment:
I did not find where you use this function

Reply:
just kept as algorithm holder